modules/nextflow/src/main/groovy/nextflow/NF.groovy

```diff
@@ -79,4 +80,9 @@ class NF {
     static boolean isTopicChannelEnabled() {
         NextflowMeta.instance.preview.topic
     }
+
+    @Memoized
+    static String bash() {
+        session().config.navigate('nextflow.defaults.bash') ?: '/bin/bash'
```

Comment:
This config scope was renamed to `workflow`, I would just call it `workflow.bash`

Reply:
let's discuss about this. I think we need to nextflow wide setting (more tied to runtime than workflow concept)